app/controllers/admin/users_controller.rb

```diff
@@ -22,8 +22,12 @@ def create
       create_params = resource_params
       create_params.delete(:is_superadmin)
       resource = resource_class.new(create_params)
-
+      resource.password = resource.generate_password
+      resource.password_confirmation = resource.password
+      resource.skip_confirmation!
+      
```

Comment:
An empty line should not contain spaces. An example of the rule - no line should have trailing spaces.